apps/stack/template.yaml

```diff
@@ -37,6 +37,12 @@ Metadata:
           default: CloudWatch Metrics
         Parameters:
           - MetricStreamFilterUri
+      - Label:
+          default: Cloudwatch Metrics Poller
+        Parameters:
+          - ObserveAwsAccountId
+          - AllowedActions
```

Comment:
why are we taking this in as a param? shouldn't we know what we need?

Reply:
great point, let me fix that.